helm/kubernetes-cluster-prep/values.schema.json

```diff
@@ -22,7 +22,7 @@
                 }
             }
         },
-        "authnk8s": {
+        "authnK8s": {
```

Comment:
Nice catch!
We definitely want this fix, but it raises a couple of issues:

1. **How the heck were all of our schema validation tests working for the kubernetes-cluster-prep chart without this fix???** I did some experimentation, and it appears that all of our schema validation for names of Kubernetes objects is superfluous. Helm has built-in validation for these names that apparently takes precedence over our entries in `values.schema.json`. The `regex` that is used for the built-in checking is also bit different than what we have (as is the maxLen). My preference would be to not include any validation for names of Kubernetes objects (anything we add can be misleading/incorrect). I'll take the blame for the churn on this one, since I think I added this stuff back in when I added the UT.

2. Adding this line breaks the `test-unit` and `test-schema` tests in kubernetes-cluster-prep. There are some nil errors that show up for some test cases. Here's what I had to change to get tests to pass:

```
diff --git a/helm/kubernetes-cluster-prep/values.yaml b/helm/kubernetes-cluster-prep/values.yaml
index ef9a2cc..a067485 100644
--- a/helm/kubernetes-cluster-prep/values.yaml
+++ b/helm/kubernetes-cluster-prep/values.yaml
@@ -34,7 +34,7 @@ authnK8s:
     # "authn-k8s-clusterrole". If 'authnK8s.clusterRole.create` is set to `false`,
     # then this is a required value. Example:
     # name: authn-k8s-clusterrole
-    name:
+    #name:
 
   serviceAccount:
     create: true
@@ -44,6 +44,6 @@ authnK8s:
     # to "authn-k8s-serviceaccount". If 'authnK8s.serviceAccount.create` is set
     # to `false`, then this is a required value. Example:
     # name: authn-k8s-serviceaccount
-    name:
+    #name:
 
 displaySamplePolicy: false
```

If you'd prefer to merge any new changes to kubernetes-cluster-prep as a separate PR, I can quickly approve. Let me know what you think about removing the validity checks for Kubernetes object names.